base/reflection.jl

```diff
@@ -542,14 +546,18 @@ code_llvm(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types)
 code_llvm_raw(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types, false)
 
 """
-    code_native([io], f, types)
+    code_native([io], f, types, [asm_variant])
```

Comment:
It looks like this signature of `code_native` including an `io` argument isn't listed in the rst manual at `doc/stdlib/base.rst`, but maybe it should be? See CONTRIBUTING.md for doc addition steps, but if you add a spot that has the signature match then do `make docs` the rest of the docstring content will get auto-populated.